radio.k0swe.kel_agent.Kel_Agent.yml

```diff
@@ -0,0 +1,45 @@
+app-id: radio.k0swe.kel_agent.Kel_Agent
```

Comment:
This app-id doesn't follow the rules as the domain doesn't exist.

Reply:
Changed to radio.k0swe.Kel_Agent. I do control https://www.k0swe.radio/.